lib/modules/manager/terraform/__fixtures__/modules.tf

```diff
@@ -62,9 +62,24 @@ module "task_definition" {
   source = "github.com/tieto-cem/terraform-aws-ecs-task-definition?ref=v0.1.0"
   name   = "mytask"
   container_definitions = [
-    "${module.container_definition.json}"]
+  "${module.container_definition.json}"]
```

Comment:
Revert

It was maybe here to check that non-traditional whitespace is still parsed correctly